src/server/pfs/server/master.go

```diff
@@ -27,6 +28,7 @@ const (
 )
 
 func (d *driver) master(ctx context.Context) {
+	ctx = auth.AsInternalUser(ctx, "pfs-master")
```

Comment:
this is the approach I've taken for metrics and pps master, this makes it appear authorized but gives no permissions